docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -103,13 +103,8 @@ When a trace is hierarchically linked to a parent trace, we say that this trace
 
 Applications that interact with the AWS JDBC Driver may or may not have already opened telemetry traces on their own. In this case, it is up to the user to determine how they want to mix both application and driver traces.
 
-This can be done using the AWS JDBC Driver's `telemetrySubmitTopLevel` property. This property allows users to choose to submit the driver traces always as top level traces, to submit them nested to the application traces, or even to not submit the driver traces at all.
-
-The `telemetrySubmitTopLevel` property allows 4 different options to choose from:
-- `ALWAYS_TOP_LEVEL` means that all the driver traces should be all forced into being top level traces. If the application has already open traces, it will not be possible for the driver traces to be top level and the driver traces therefore will not be submitted.
-- `ALWAYS_NESTED` indicates that the user wants the driver traces to always be nested into their application traces. That will happen unless there are no open application traces when the driver is running. In that case no driver traces will be collected or submitted.
-  -`TOP_LEVEL` is a strategy where driver traces are submitted as top level traces, unless there is already an application open trace. If there is already an open trace, the driver traces will identify it and be submitted as nested traces with the application trace as parent.
-- `NO_TRACE` indicates that no traces will be submitted to Amazon XRay.
+This can be done using the AWS JDBC Driver's `telemetrySubmitTopLevel` property. This property allows users to choose to submit the driver traces always as top level traces or to submit them nested to the application traces.
+The default value is set to `false`, which means the driver traces to always be nested into their application traces. That will happen unless there are no open application traces when the driver is running. In that case no driver traces will be collected or submitted. When the property is set to `true`, all the driver traces would be submitted top level traces. If the application has already open traces, it will not be possible for the driver traces to be top level and the driver traces therefore will not be submitted.
```

Comment:
```suggestion
The default value is set to `false`, which means the driver traces will always be nested into the application traces. If there are no open application traces when the driver is running, the driver traces will not be nested into the application traces and no driver traces will be collected or submitted. When the property is set to `true`, all the driver traces will be submitted as top level traces. If the application already has open traces, it will not be possible for the driver traces to be top level, and the driver traces will not be submitted.
```